### ISSUE TITLE
Dataset problems itemself

### ISSUE THREAD

**Issue by wheltz** [talk2car/Talk2Car#10]
There are many examples in the existing talk2car data set where the command reference is unclear (a command may actually have multiple reference objects)， and there are many more such examples.
>![image](https://github.com/talk2car/Talk2Car/assets/57028913/64397cea-5c7c-4ef2-a48a-f7a7a4868ba2)
![image](https://github.com/talk2car/Talk2Car/assets/57028913/5f581b98-7c17-4253-9cd7-c03956fb4be5)
![image](https://github.com/talk2car/Talk2Car/assets/57028913/34e6031e-6136-48f4-81e5-3ff9faf01881)

Some images are so unclear that I doubt whether manual annotation can be successful
> ![image](https://github.com/talk2car/Talk2Car/assets/57028913/3716803a-3c0d-4aa5-84f6-1abb50f31172)
![image](https://github.com/talk2car/Talk2Car/assets/57028913/31534279-4dbe-4497-b050-f74aecc16af7) 

even some annotation errors.
![image](https://github.com/talk2car/Talk2Car/assets/57028913/1b8dd786-b24d-4ba3-802b-4b39d9a2fad2) 
My data set comes entirely from the link under your warehouse. I don’t know if you have discovered this problem and have prepared a cleaned data set, but it is really too painful for me to clean this data.

**Comment by ThierryDeruyttere** [comment]
yes, this is the result of doing mass annotations with mechanical turkers. We did a manual pass over the data but it did not catch 100% of the mistakes. If you look at other well-known datasets you will see that they are also not 100% correct. Quality assurance on these big datasets is not an easy task.

When creating the annotations, people could cycle through ground truth bounding boxes with the class of each object. So every bounding box you see is a converted ground truth 3D bounding box from the nuScenes dataset to 2D.

The black image is a challenging one that I agree on. But this is a good example of what self-driving vehicles need to be able to work in. The camera sensor is probably not optimal in these kind of conditions but the dataset also provides radar/Lidar clouds that should be beneficial in these situations.

The last image is actually "correct" as there is a black car on the left but indeed you do not see it in this frame anymore but for some reason the bounding box still remained in this image.

And yes we are aware of this "problem" but we still believe that the biggest part of the data is of high quality. So no we do not have a cleaned data set. What we released is the dataset that we used in all our papers.
